plugins/dependencies/src/main/kotlin/de/fayard/refreshVersions/internal/PluginConfig.kt

```diff
@@ -0,0 +1,79 @@
+package de.fayard.internal
+
+
+@Suppress("unused")
+object PluginConfig {
```

Comment:
Why is this public instead of `internal`?

Reply:
Good points, I will take care of it once https://github.com/jmfayard/refreshVersions/pull/281 is merged